test/wallet.test.ts

```diff
@@ -253,4 +268,38 @@ describe('Wallet - Transaction builder', () => {
       });
     });
   });
+
+  describe('FetchAndUnblindTx function', () => {
```

Comment:
I would add another test case that try sto spend an input to another address, so we test (and document) also the unblindinding of a prevout

Reply:
@louisinger